app/controllers/items_controller.rb

```diff
@@ -1,3 +1,6 @@
+require "amazon_product_api"
+include AWSAPIs
```

Comment:
Are we including this module so that `AmazonProductAPI` can be used in a non-module-scoped way on line 69?